cookbook/doctrine/registration_form.rst

```diff
@@ -200,8 +200,8 @@ Start by creating a simple class which represents the "registration"::
 
 Next, create the form for this ``Registration`` model::
```

Comment:
I kept "Model" for the `Registration` class because it isn't a doctrine entity like the `User` class.